robot_skills/src/robot_skills/base.py

```diff
@@ -134,7 +134,7 @@ def getPlan(self, position_constraint):
     def checkPlan(self, plan):
         try:
             resp = self._check_plan_client(plan)
-        except:
+        except Exception:
             rospy.logerr("Could not check plan, is the global planner running?")
```

Comment:
Please also print the error, especially since this is a very broad exception class.

If possible, also catch a more narrow exception

Reply:
Done